downstream/modules/platform/proc-aap-controller-backup.adoc

```diff
@@ -41,3 +41,16 @@ $ df -h | grep "/var/lib/pgsql/data"
 . Click btn:[Create].
 +
 A backup tarball of the specified deployment is created and available for data recovery or deployment rollback. Future backups are stored in separate tar files on the same pvc.
+
+.Verification
+. Log in to Red Hat *{OCP}*
+. Navigate to menu:Operators[Installed Operators].
+. Select the {OperatorPlatform} installed on your project namespace.
+. Select the *AutomationControllerBackup* tab.
+. Select the backup resource you want to verify.
+. Scroll to *Conditions* and check that the *Successful* status is `True`.
++
+[NOTE]
+====
+If *Successful* is `False`, the backup has failed. Check the automation controller operator logs for the error to fix the issue.
```

Comment:
should we use the attribute for automation controller here? Or is this acceptable with automation controller operator?

Reply:
This is fixed now, thanks!